packages/cli/src/commands/balance.ts

```diff
@@ -0,0 +1,43 @@
+import balance from '../scripts/balance';
+import { promptIfNeeded, networksList, InquirerQuestions } from '../prompts/prompt';
+import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesInitializer';
+import Session from '../models/network/Session';
+
+const name: string = 'balance';
+const signature: string = `${name} [address]`;
+const description: string = 'query the balance of the specified account';
+
+const register: (program: any) => any = (program) => program
+  .command(signature, undefined, { noHelp: true })
+  .usage('[options]')
+  .description(description)
+  .option('--erc20 <contractAddress>', 'specify an ERC20 token address or leave empty to query ETH balance')
```

Comment:
```suggestion
  .option('--erc20 <contractAddress>', 'query the balance of an ERC20 token instead of ETH')
```
"Leave empty" seems to indicate that the value of the flag should be left empty, not that the user should not set the flag at all.